files/default/neutron-ha-tool.py

```diff
@@ -54,10 +54,13 @@
 
 def parse_args():
     # ensure environment has necessary items to authenticate
-    for key in ['OS_USERNAME', 'OS_PASSWORD', 'OS_AUTH_URL',
-                'OS_ENDPOINT_TYPE']:
+    for key in ['OS_USERNAME', 'OS_AUTH_URL', 'OS_REGION_NAME']:
```

Comment:
Thanks for the commit message update! Ahh, I misunderstood - we're not actually dropping `OS_ENDPOINT_TYPE`, but merely making it optional.  So the commit message is still slightly misleading.